examples/eqPly/config.cpp

```diff
@@ -90,6 +90,9 @@ bool Config::init()
     _loadModels();
     _registerModels();
 
+    if (_initData.centerCamera())
+        _frameData.setCameraPosition(eq::Vector3f::zero());
```

Comment:
Isn't the default camera position (0, 0, 0)? Shouldn't this logic go to wherever the camera is set to something different?

Reply:
It isn't, it's z=-2.0, that's what you get when you press the space bar (it goes to (0,0,0) if you press it a second time). There was also a default rotation which I removed in the second commit. I would be possible to invert the logic a bit, but we still need to go to z=-2.0 by default.